code/modules/admin/verbs/randomverbs.dm

```diff
@@ -761,30 +762,11 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	var/confirm = alert(src, "You sure?", "Confirm", "Yes", "No")
 	if(confirm != "Yes") return
 
-	var/choice
-	if(emergency_shuttle.auto_recall)
-		choice = input("The shuttle will just return if you call it. Call anyway?") in list("Confirm", "Cancel")
-		if(choice != "Confirm")
-			return
-
-	if(emergency_shuttle.no_escape)
-		choice = input("The shuttle will not be able to leave if you call it. Call anyway?") in list("Confirm", "Cancel")
-		if(choice != "Confirm")
-			return
-
-
-
-	choice = input("Is this an emergency evacuation or a crew transfer?") in list("Emergency", "Crew Transfer")
-	if (choice == "Emergency")
-		var/reason = input(usr, "Optional: Please enter the reason for calling the shuttle.", "Shuttle Call Reason.","") as text|null
-		emergency_shuttle.call_evac(reason)
-	else
-		emergency_shuttle.call_transfer()

```

Comment:
transfer shuttles aren't a thing with -tg- shuttles
not particularly inclined to bother for a bit of fluff either